packages/client/src/runtime/core/extensions/MergedExtensionsList.ts

```diff
@@ -47,7 +47,7 @@ class MergedExtensionsListNode {
     })
   }
 
-  getAllQueryCallbacks(jsModelName: string, operation: string) {
+  getAllQueryCallbacks(jsModelName: string | '*', operation: string) {
```

Comment:
Can we use some other way to indicate top level operation? I'd read `getAllQueryCallbacks('*')` as "query callbacks for all models"

I think, making `jsModelName` nullable would work better in this case.

Reply:
Yes, we can, I think this was done for the caching logic, so we had to encode `undefined` into something else since it expects a string and would probably be incorrect to stringify `undefined` as `"undefined"` as that could match a model name (unlikely but possible). So maybe `"<undefined>"` works best? Or something else?

Reply:
Could be `$topLevel` or `$none` then.

Reply:
Changed to `$none`